setup.cfg

```diff
@@ -48,13 +48,14 @@ project_urls =
 [options]
 packages = find:
 
-python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*
+python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
```

Comment:
This change doesn't seem directly related, and may have other implications. I'm not against dropping support for Python 3.4, but it should be advertized in another PR with a proper title (that could add official support for 3.9 in the same time?)

Reply:
importlib_resources is not supported by Python 3.4, which is why this was introduced, happy to drop this though

Reply:
Please don't drop it if it's needed. My proposal was to make it official (to other people reviewing yamllint) in a dedicated PR (and why not add official support for 3.9 in the same time).

Reply:
Ok, I added a new PR here to add support for Python 3.9 and drop support for Python 3.4: https://github.com/adrienverge/yamllint/pull/345